### PR TITLE
kubernetes-cn-translation-update

### PR DESCRIPTION
**Pull request by markthink**
The previous translation file was properly updated to be submitted with a smaller pr granularity.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4182)

### CODE REVIEW

**Comment by chenopis** [comment]
@markthink So who can do the doc and tech reviews? Since this is in the /cn directory, I'm fine w/ merging it whenever you give me the approval. Thx

**Comment by xinzhangcmu** [comment]
@chenopis Per discussion with Dan Kohn @dankohn from CNCF, it'd be great if another native Chinese speaker (not from Caicloud to avoid bias) could kindly do the review and give the approval. 

@wangtt03 from Microsoft wanna give a hand here?

**Comment by markthink** [comment]
@chenopis  In order to advance the progress, can let @wangtt03  help review ?

**Comment by markthink** [comment]
@chenopis  What @xinzhangcmu said [呲牙]

**Comment by chenopis** [comment]
Ok, sounds good. Just let me know when it's been approved and is ready to be merged. Thx

**Comment by markthink** [comment]
@chenopis thanks

**Comment by xingzhou** [comment]
I will also take a look at the translation:)

File: cn/docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ assignees:
...
-Kubernetes 是一个 [开源的容器调度平台，它可以自动化应用容器的部署、扩展和操作](http://www.slideshare.net/BrianGrant11/wso2con-us-2015-kubernetes-a-platform-for-automating-deployment-scaling-and-operations) 可以跨主机集群, 提供以容器为中心的基础架构。
+Kubernetes 是一个跨主机集群的 [开源的容器调度平台，它可以自动化应用容器的部署、扩展和操作](http://www.slideshare.net/BrianGrant11/wso2con-us-2015-kubernetes-a-platform-for-automating-deployment-scaling-and-operations) , 提供以容器为中心的基础架构。
 
 使用 Kubernetes, 您可以快速高效地响应客户需求:
 
  - 快速、可预测地部署您的应用程序
  - 拥有即时扩展应用程序的能力
  - 不影响现有业务的情况下，无缝地发布新功能。
- - 优化您的硬件资源，降低您的拥有成本
+ - 优化您的硬件资源，降低您的所需成本
```
Comment (xingzhou) [inline cn/docs/concepts/overview/what-is-kubernetes.md]: Maybe just simplified as "优化硬件资源，降低成本"？

File: cn/docs/concepts/overview/what-is-kubernetes.md [hunk 2]
```diff
@@ -87,11 +87,11 @@ Kubernetes 满足了生产中运行应用程序的许多常见的需求，例如
...
+Kubernetes 提供了很多的功能，总会有新的场景受益于新特性。它可以简化应用程序的工作流，加快开发速度。被大家认可的应用编排通常需要有较强的自动化能力。这就是为什么 Kubernetes 被设计作为构建组件和工具的生态系统平台，以便更轻松地部署、扩展和管理应用程序。
 
 [Label](/docs/user-guide/labels/) 允许用户按照自己的方式组织管理对应的资源。 [注解](/docs/user-guide/annotations/) 使用户能够以自定义的描述信息来修饰资源，以适用于自己的工作流，并为管理工具提供检查点状态的简单方法。
 
-此外，[Kubernetes 控制面](/docs/admin/cluster-components) 是构建在相同的 [APIs](/docs/api/) 上面，开发员人和用户都可以用。用户可以编写自己的控制器， [调度器](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/scheduler.md)等等，如果这么做，根据新加的[自定义 API](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/extending-api.md) ，可以扩展当前的通用 [CLI 命令行工具](/docs/user-guide/kubectl-overview/)。
+此外，[Kubernetes 控制面](/docs/admin/cluster-components) 是构建在相同的 [APIs](/docs/api/) 上面，开发人员和用户都可以用。用户可以编写自己的控制器， [调度器](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/scheduler.md)等等，如果这么做，根据新加的[自定义 API](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/extending-api.md) ，可以扩展当前的通用 [CLI 命令行工具](/docs/user-guide/kubectl-overview/)。
```
Comment (xingzhou) [inline cn/docs/concepts/overview/what-is-kubernetes.md]: For some of the name of K8s component, I think we can pick one of the following translation options:
• Keep the original English word, e.g. Scheduler, Controller
• Translate the name into Chinese, but put the original English word in the following bracket, e.g. 调度器（Scheduler）
• Translate the name into Chinese directly
IMO, the second option is better

Comment (xingzhou) [inline cn/docs/concepts/overview/what-is-kubernetes.md]: Besides, some of the translations of `Controll Plane` is `控制平面`，some similar translation like `data plane` and `application plane`

File: cn/docs/tutorials/kubernetes-basics/index.html [hunk 3]
```diff
@@ -77,7 +77,7 @@
           <div class="thumbnail">
             <a href="/docs/tutorials/kubernetes-basics/scale-intro/"><img src="./public/images/module_05.svg?v=1469803628347" alt=""></a>
             <div class="caption">
-              <a href="scale-intro/"><h5>5. 应用可伸缩</h5></a>
+              <a href="scale-intro/"><h5>5. 应用可扩展</h5></a>
```
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/index.html]: how about "垂直扩展应用程序"?
and also for those specific technical word, like `scale up` here, can we both keep the Chinese translation together with the original English word? e.g. `5. 垂直扩展应用程序(scale up)`

File: cn/docs/tutorials/kubernetes-basics/cluster-intro.html [hunk 4]
```diff
@@ -31,12 +31,12 @@
...
 				<p>
-				<b>Kubernetes 用于协调高度可用的计算机集群，这些计算机群集被连接作为单个单元工作。</b> Kubernetes 中的抽象允许您将容器化的应用程序部署到集群，而不必专门将其绑定到单个计算机。为了利用这种新的部署模型，应用程序需要以将它们与各个主机分离的方式打包: 它们需要被容器化。容器化应用程序比过去的部署模型更灵活和可用，其中应用程序直接安装到特定机器上，作为深入集成到主机中的软件包。 <b>Kubernetes 以更有效的方式自动化、跨集群的容器应用程序的分发和调度。</b> Kubernetes 是一个 <a href="https://github.com/kubernetes/kubernetes">开源</a> 平台，为生产环境准备的。
+				<b>Kubernetes 用于协调高度可用的计算机集群，这些计算机群集被连接作为单个单元工作。</b> Kubernetes 的抽象性允许您将容器化的应用程序部署到集群，而不必专门将其绑定到单个计算机。为了利用这种新的部署模型，应用程序需要以将它们与各个主机分离的方式打包: 它们需要被容器化。容器化应用程序比过去的部署模型更灵活和可用，其中应用程序直接安装到特定机器上，作为深入集成到主机中的软件包。 <b>Kubernetes 在一个集群上以更有效的方式自动分发和调度容器应用程序。</b> Kubernetes 是一个 <a href="https://github.com/kubernetes/kubernetes">开源</a> 平台，并且已经准备好了帮助生产。
 				</p>
 				<p>Kubernetes 集群由两种类型的资源组成:
 					<ul>
-			            <li>一个 <b>Master</b> 调度节点</li>
-			            <li><b>Nodes</b> 应用程序实际运行的地方</li>
+			            <li>一个 <b>Master</b> 调度集群</li>
+			            <li><b>节点</b> 是应用程序实际运行的地方</li>
```
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/cluster-intro.html]: same comments here, can we both keep the English and Chinese for specific technical words here, like Nodes

File: cn/docs/tutorials/kubernetes-basics/cluster-intro.html [hunk 5]
```diff
@@ -31,12 +31,12 @@
 			<div class="col-md-8">
 				<h3>Kubernetes 集群</h3>
 				<p>
-				<b>Kubernetes 用于协调高度可用的计算机集群，这些计算机群集被连接作为单个单元工作。</b> Kubernetes 中的抽象允许您将容器化的应用程序部署到集群，而不必专门将其绑定到单个计算机。为了利用这种新的部署模型，应用程序需要以将它们与各个主机分离的方式打包: 它们需要被容器化。容器化应用程序比过去的部署模型更灵活和可用，其中应用程序直接安装到特定机器上，作为深入集成到主机中的软件包。 <b>Kubernetes 以更有效的方式自动化、跨集群的容器应用程序的分发和调度。</b> Kubernetes 是一个 <a href="https://github.com/kubernetes/kubernetes">开源</a> 平台，为生产环境准备的。
+				<b>Kubernetes 用于协调高度可用的计算机集群，这些计算机群集被连接作为单个单元工作。</b> Kubernetes 的抽象性允许您将容器化的应用程序部署到集群，而不必专门将其绑定到单个计算机。为了利用这种新的部署模型，应用程序需要以将它们与各个主机分离的方式打包: 它们需要被容器化。容器化应用程序比过去的部署模型更灵活和可用，其中应用程序直接安装到特定机器上，作为深入集成到主机中的软件包。 <b>Kubernetes 在一个集群上以更有效的方式自动分发和调度容器应用程序。</b> Kubernetes 是一个 <a href="https://github.com/kubernetes/kubernetes">开源</a> 平台，并且已经准备好了帮助生产。
```
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/cluster-intro.html]: Suggest "并且已经准备好了帮助生产" -> “可以满足生产环境的需要”

File: cn/docs/tutorials/kubernetes-basics/deploy-intro.html [hunk 6]
```diff
@@ -27,26 +27,26 @@
 			<div class="col-md-8">
 				<h3>Kubernetes 部署</h3>
 				<p>
-				一旦运行了 Kubernetes 集群，您可以在其上部署容器化应用程序。为此，您可以创建一个 Kubernetes <b>Deployment</b>。Deployment 负责创建和更新应用程序实例。创建 Deployment 后， Kubernetes master 会将 Deployment 创建的应用程序实例调度到集群中的各个节点。
+				一旦运行了 Kubernetes 集群，您可以在其上部署容器化应用程序。为此，您可以创建一个 Kubernetes <b>部署</b>。部署负责创建和更新应用程序实例。创建部署 后， Kubernetes master 会将部署创建的应用程序实例调度到集群中的各个节点。
```
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/deploy-intro.html]: Same comments here, for those specific K8s names, like deployment, I suggest keep original English word, or at least keep English word together with Chinese translation. Similar situations like pod/StatefulSet/Job and such like

File: cn/docs/tutorials/kubernetes-basics/explore-intro.html [hunk 7]
```diff
@@ -34,23 +34,23 @@
...
 						<li>Pods</li>
-						<li>Nodes</li>
+						<li>节点</li>
 						<li>Kubectl 主要命令</li>
 					</ul>
 				</div>
 				<div class="content__box content__box_fill">
 						<p><i>
-							Pod 是一组或多个应用程序容器 (例如 Docker 或 rkt)，包含共享存储 (卷)，IP 地址以及有关如何运行它们的信息。
+							Pod 是一组一个或多个应用程序容器 (例如 Docker 或 rkt)，包含共享存储 (卷)，IP 地址以及有关如何运行它们的信息。
```
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/explore-intro.html]: Pod包含一个或者多个应用程序容器
or
Pod是由一个或者多个应用程序容器构成的

File: cn/docs/tutorials/kubernetes-basics/expose-intro.html [hunk 8]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 使用服务让您的应用程序外部可见
+title: 使用服务公开您的应用程序
```
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/expose-intro.html]: I'm not sure whether use "发布" is better?

File: cn/docs/tutorials/kubernetes-basics/expose-intro.html [hunk 9]
```diff
@@ -15,98 +15,77 @@
...
-      	<h3>目标</h3>
+			<div class="col-md-8">
+    		<h3>目标</h3>
 				<ul>
-					<li>了解 Kubernetes 服务</li>
-					<li>了解 Kubernetes 标签</li>
-					<li>应用程序在 Kubernetes 外部可见</li>
+					<li>了解 Kubernetes 中的服务</li>
+					<li>了解标签和标签选择器对象如何与服务相关联</li>
+					<li>使用服务在 Kubernetes 群集外公开应用程序</li>
```
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/expose-intro.html]: how about "在Kubernetes集群外公开应用程序"？

File: cn/docs/tutorials/kubernetes-basics/scale-intro.html [hunk 10]
```diff
@@ -19,14 +19,15 @@
...
       	<h3>目标</h3>
 				<ul>
-					<li>使用 kubectl 缩放应用程序</li>
+					<li>使用 kubectl 伸缩应用程序</li>
 				</ul>
 			</div>
 
 			<div class="col-md-8">
-       <h3>缩放应用程序</h3>
+       <h3>伸缩应用程序</h3>
```
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/scale-intro.html]: how about “应用程序伸缩”？

File: cn/docs/tutorials/kubernetes-basics/update-intro.html [hunk 11]
```diff
@@ -26,9 +26,9 @@
 			<div class="col-md-8">
 			<h3>更新应用程序</h3>
 
-			<p>用户期望应用程序始终可用，并且开发人员有望每天部署新版本。这就是 Kubernetes 的滚动更新。 <b>Rolling updates</b> 允许通过使用新的 Pods 实例逐个更新来实现零停机的更新部署。新的 Pods 会被调度到可用资源的 Node 节点上。</p>
+			<p>用户期望应用程序始终可用，并且希望开发人员每天部署新版本。在 Kubernetes 上这通过滚动更新达成。 <b>Rolling updates</b> 允许通过使用新的 Pods 实例逐个更新来实现零停机的部署更新。新的 Pods 会被调度到可用资源的 Node 节点上。</p>
```
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/update-intro.html]: similar comments， for specific technical names, like rolling update/rollout, I think at least we need to keep the English words

File: cn/docs/tutorials/object-management-kubectl/object-management.md [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+title: Kubernetes 对象管理
+redirect_from:
+- "/docs/concepts/tools/kubectl/object-management-overview/"
+- "/docs/concepts/tools/kubectl/object-management-overview.html"
+- "/docs/user-guide/working-with-resources/"
+- "/docs/user-guide/working-with-resources.html"
+---
+
+{% capture overview %}
+`kubectl` 命令行工具支持 Kubernetes 对象几种不同的创建和管理方法。本文档概述了不同的方法.
```
Comment (xingzhou) [inline cn/docs/tutorials/object-management-kubectl/object-management.md]: how about 
`kubectl` 命令行工具支持 Kubernetes 对象几种不同的创建和管理方法。本文档简要介绍了这些方法.

File: cn/docs/tutorials/object-management-kubectl/object-management.md [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+{% endcapture %}
+
+{% capture body %}
+
+## 管理技巧
+
+**警告:** Kubernetes 对象应该只使用一种技术进行管理。混合使用不同的技术，会导致相同对象出现未定义的行为。
+
+| 管理技术             | 操作         |推荐环境 | 支持撰写  | 学习曲线 |
```
Comment (xingzhou) [inline cn/docs/tutorials/object-management-kubectl/object-management.md]: 操作对象

File: cn/docs/tutorials/object-management-kubectl/object-management.md [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+|----------------------------------|----------------------|------------------------|--------------------|----------------|
+| 命令式的方式              | 活动对象         | 开发项目   | 1+                 | 最低         |
+| 命令式对象配置  | 单文件     | 生产项目    | 1                  | 中等       |
+| 声明式对象配置 | 文件目录 | 生产项目    | 1+                 | 最高        |
+
+## 命令式的方式
```
Comment (xingzhou) [inline cn/docs/tutorials/object-management-kubectl/object-management.md]: 命令式的命令？

File: cn/docs/tutorials/object-management-kubectl/object-management.md [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+ - 命令不与变更审核流程整合。
+ - 命令不提供与更改相关联的审计跟踪。
+ - 除了活动对象之外，命令不提供记录来源。
+ - 命令不提供用于创建新对象的模板。
+
+## 命令式对象配置
+
+在命令式对象配置中，`kubectl` 命令指定操作(创建，替换等)，可选标志和至少一个文件名称。指定的文件必须包含对象的完整定义以 YAML 或 JSON 格式。
+
+请参阅[资源参考](https://kubernetes.io/docs/resources-reference/v1.6/)
```
Comment (xingzhou) [inline cn/docs/tutorials/object-management-kubectl/object-management.md]: 参考资源

**Comment by xingzhou** [comment]
The translation looks good, put some small comments on the PR, @markthink, FYI

**Comment by markthink** [comment]
@xingzhou I am updating it，thank you.

File: cn/docs/concepts/overview/what-is-kubernetes.md (hunk 1 above)
Comment (markthink) [inline cn/docs/concepts/overview/what-is-kubernetes.md]: Can be simplified

File: cn/docs/concepts/overview/what-is-kubernetes.md (hunk 2 above)
Comment (markthink) [inline cn/docs/concepts/overview/what-is-kubernetes.md]: Your advice is very good.

File: cn/docs/tutorials/kubernetes-basics/index.html (hunk 3 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/index.html]: @xingzhou Scale up has the meaning of enlargement, my opinion is to keep the original translation..e.g *5. 应用可扩展*

Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/index.html]: yeap, but here I think we need to differentiate `scale up` and `scale out`, maybe we can use the translation but keep the English word `scale up` in the bracket

File: cn/docs/tutorials/kubernetes-basics/cluster-intro.html (hunk 4 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/cluster-intro.html]: @xingzhou  How about this translation？
<li>一个 <b>Master</b> 是集群的协调节点</li>
<li><b>Nodes</b> 是应用程序实际运行的工作节点</li>

File: cn/docs/tutorials/kubernetes-basics/cluster-intro.html (hunk 5 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/cluster-intro.html]: good

File: cn/docs/tutorials/kubernetes-basics/index.html (hunk 3 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/index.html]: @xingzhou Okay, I keep the original

File: cn/docs/tutorials/kubernetes-basics/cluster-intro.html (hunk 4 above)
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/cluster-intro.html]: Sure, good to me:) thanks

File: cn/docs/tutorials/kubernetes-basics/deploy-intro.html (hunk 6 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/deploy-intro.html]: @xingzhou How about this.
<p>
				一旦运行了 Kubernetes 集群，您可以在其上部署容器化应用程序。为此，您可以创建一个 Kubernetes <b>Deployment</b>。Deployment 负责创建和更新应用程序实例。创建 Deployment 后， Kubernetes master 会将 Deployment 创建的应用程序实例调度到集群中的各个节点。
				</p>

File: cn/docs/tutorials/kubernetes-basics/explore-intro.html (hunk 7 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/explore-intro.html]: @xingzhou I choose
Pod是由一个或者多个应用程序容器构成的

File: cn/docs/tutorials/kubernetes-basics/deploy-intro.html (hunk 6 above)
Comment (xingzhou) [inline cn/docs/tutorials/kubernetes-basics/deploy-intro.html]: yeap, thanks:)

File: cn/docs/tutorials/kubernetes-basics/expose-intro.html (hunk 8 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/expose-intro.html]: @xingzhou  I agree with your comments expose translated into it (发布)
使用服务发布您的应用程序

File: cn/docs/tutorials/kubernetes-basics/expose-intro.html (hunk 9 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/expose-intro.html]: @xingzhou How about this.
通过 Service 在 Kubernetes 集群外发布应用程序？

File: cn/docs/tutorials/kubernetes-basics/scale-intro.html (hunk 10 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/scale-intro.html]: good

Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/scale-intro.html]: good

File: cn/docs/tutorials/kubernetes-basics/update-intro.html (hunk 11 above)
Comment (markthink) [inline cn/docs/tutorials/kubernetes-basics/update-intro.html]: @xingzhou  Similar to this
用户期望应用程序始终可用，并且希望开发人员每天部署新版本。在 Kubernetes 上这通过滚动更新(Rolling updates)达成。

File: cn/docs/tutorials/object-management-kubectl/object-management.md (hunk 12 above)
Comment (markthink) [inline cn/docs/tutorials/object-management-kubectl/object-management.md]: good

File: cn/docs/tutorials/object-management-kubectl/object-management.md (hunk 13 above)
Comment (markthink) [inline cn/docs/tutorials/object-management-kubectl/object-management.md]: @xingzhou sorry.I do not quite understand where I need to update

File: cn/docs/tutorials/object-management-kubectl/object-management.md (hunk 14 above)
Comment (markthink) [inline cn/docs/tutorials/object-management-kubectl/object-management.md]: @xingzhou Command itself contains the meaning of the composition of the order, so there is no such translation.

File: cn/docs/tutorials/object-management-kubectl/object-management.md (hunk 15 above)
Comment (markthink) [inline cn/docs/tutorials/object-management-kubectl/object-management.md]: good

**Comment by markthink** [comment]
@xingzhou @chenopis Please take another look,thanks!

**Comment by chenopis** [comment]
@markthink Unfortunately, I can't read Chinese. :( But just let me know when you think it's ready to be merged.

**Comment by markthink** [comment]
@chenopis  I think it can be merge

**Comment by xingzhou** [comment]
I think the doc is good for merge now, and for some tiny issues, we can always update in the future.
For Chinese translation, it's better to setup a team in the community, that would be easier for review and merge. /cc @chenopis @markthink and @xinzhangcmu

**Pull request merged**